### PR TITLE
Add Authorization for our React Components

### PR DESCRIPTION
**Pull request by franzejr**

### CODE REVIEW

File: src/output_lib/actions/RespondToFormActions.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Form, FormSubmission } from "../api";
+import { createForm, FormSubmission } from "../api";
```
Comment (knewter) [inline src/output_lib/actions/RespondToFormActions.js]: I think `createFormApi` is a better name. And won't we need this for `createFormSubmissionApi` as well eventually?

Comment (franzejr) [reply src/output_lib/actions/RespondToFormActions.js]: @knewter, done!

File: src/output_lib/api.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // @flow
 
-import Form from "./api/Form";
+import createFormApi from "./api/Form";
 import Section from "./api/Section";
-import FormSubmission from "./api/FormSubmission";
+import createFormSubmissionApi from "./api/FormSubmission";
 
-export { Form, Section, FormSubmission };
+export { createFormApi, Section, createFormSubmissionApi };
```
Comment (knewter) [inline src/output_lib/api.js]: won't we need the same thing for Section? Do we even use this anywhere?

Comment (franzejr) [reply src/output_lib/api.js]: I've just removed that! We were not using it!

**Pull request merged**